static_compress/mixin.py

```diff
@@ -1,11 +1,73 @@
+import os
+import errno
+from os.path import getatime, getctime, getmtime
+
+from django.core.exceptions import ImproperlyConfigured
+
 from . import compressors
 
 __all__ = ['CompressMixin']
 
 
+DEFAULT_METHODS = ['gz', 'br']
+METHOD_MAPPING = {
+	'gz': compressors.ZlibCompressor,
+	'br': compressors.BrotliCompressor,
+	'gz+zop': compressors.ZopfliCompressor,
```

Comment:
I prefer `gz+zopfli` so it is Google-able if someone found this in the settings without knowing what it is